src/sage/all.py

```diff
@@ -191,20 +193,27 @@
 
 from sage.misc.persist import register_unpickle_override
 register_unpickle_override('sage.categories.category', 'Sets', Sets)
-register_unpickle_override('sage.categories.category_types', 'HeckeModules', HeckeModules)
-register_unpickle_override('sage.categories.category_types', 'Objects', Objects)
-register_unpickle_override('sage.categories.category_types', 'Rings', Rings)
-register_unpickle_override('sage.categories.category_types', 'Fields', Fields)
-register_unpickle_override('sage.categories.category_types', 'VectorSpaces', VectorSpaces)
-register_unpickle_override('sage.categories.category_types', 'Schemes_over_base', sage.categories.schemes.Schemes_over_base)
-register_unpickle_override('sage.categories.category_types', 'ModularAbelianVarieties', ModularAbelianVarieties)
+register_unpickle_override('sage.categories.category_types', 'HeckeModules',
+                           HeckeModules)
+register_unpickle_override('sage.categories.category_types', 'Objects',
+                           Objects)
+register_unpickle_override('sage.categories.category_types', 'Rings',
+                           Rings)
+register_unpickle_override('sage.categories.category_types', 'Fields',
+                           Fields)
+register_unpickle_override('sage.categories.category_types', 'VectorSpaces',
+                           VectorSpaces)
+register_unpickle_override('sage.categories.category_types', 'Schemes_over_base',
+                           sage.categories.schemes.Schemes_over_base)
+register_unpickle_override('sage.categories.category_types', 'ModularAbelianVarieties',
+                           ModularAbelianVarieties)
 register_unpickle_override('sage.libs.pari.gen_py', 'pari', pari)
 
 # Cache the contents of star imports.
 sage.misc.lazy_import.save_cache_file()
 
 
-### Debugging for Singular, see trac #10903
+# Debugging for Singular, see trac #10903
 # from sage.libs.singular.ring import poison_currRing
 # sys.settrace(poison_currRing)
 
```

Comment:
The first line is a comment to the following commented code lines.

Reply:
```
# # Debugging for Singular, see trac #10903
```
?

Reply:
it is "Issue"... How about "##" instead of "####"?

Reply:
come on, please spare my time and yours.

Reply:
OK. Not a big issue.